apps/directory/src/stores/biobanksStore.js

```diff
@@ -199,6 +193,13 @@ export const useBiobanksStore = defineStore("biobanksStore", () => {
       .flatMap((collection) => collection.sub_collections).length;
   });
 
+  const biobankCardsCollectionCount = computed(() => {
+    const totalCount = biobankCards.value
+      .filter((bc) => bc.collections)
+      .flatMap((biobank) => biobank.collections).length;
```

Comment:
I don't mind to do it this way, but this is not something the graphql query can easily produce?

Reply:
Wouldn't it be easier to count collections and subcollections by checking if the parent is set?

Reply:
I'd say that would be on par, since it would need more data retrieval and similar filtering. This was easier to implement with the existing code. If you'd rather have it changed, let me know.

Reply:
If that's the case then leave it as is. No need to change.